apollo-router/src/services/snapshots/apollo_router__services__supergraph_service__tests__deferred_fragment_bounds_nullability-2.snap

```diff
@@ -0,0 +1,129 @@
+---
+source: apollo-router/src/services/supergraph_service.rs
+expression: stream.next_response().await.unwrap()
+---
+{
+  "hasNext": false,
+  "incremental": [
+    {
+      "data": null,
+      "path": [
+        "currentUser",
+        "activeOrganization",
+        "suborga",
+        0
+      ],
+      "extensions": {
+        "valueCompletion": [
```

Comment:
unrelated to the issue you're fixing, but looking at this, the `valueComppletion` errors are not properly dispatched to each element of the `incremental` array, they are copied each time :/

Reply:
I've filed #2185 to capture that.

Reply:
Thanks